src/lib/votes.ts

```diff
@@ -0,0 +1,73 @@
+import { atomFamily, getState, selectorFamily } from "atom.io"
+import { parseJson, stringifyJson } from "atom.io/json"
+import { doc, onSnapshot, setDoc } from "firebase/firestore"
+
+import type { ActualVote, SerializedVote } from "../types"
+import { myselfSelector } from "./auth"
+import { db } from "./firebase"
+
+const serializedVoteAtomsGate: Record<string, boolean> = {} // AtomIO calls onSet on setSelf, which I'm blocking using this.
```

Comment:
1. this comment doesn't make any sense to me. Need quite a bit more explanation here.
2. You're using "I" in this comment, which is fine, but my preference is for comments like this to be signed and dated.
```suggestion
/* AtomIO calls `onSet` on `setSelf`, which I'm blocking using this.
 * This is because of X, Y, and Z reasons. -@jeremybanka, Aug 2024
 */
const serializedVoteAtomsGate: Record<string, boolean> = {} 
```